Library/Formula/elm.rb

```diff
@@ -0,0 +1,23 @@
+class Elm < Formula
+  homepage "http://elm-lang.org"
+  url "https://github.com/kevva/elm-bin/raw/v1.4.1/vendor/osx/elm-platform-osx.tar.gz"
```

Comment:
Is this an official link? I couldn't find reference to it anywhere on the website.